docs/docs/ecosystem/bugbounty.md

```diff
@@ -0,0 +1,61 @@
+---
+id: bugbounty
+title: Bug Bounty Program
+---
+
+:::caution Disclaimer
+
+We are currently in the process of setting up a disclosure platform. Until that process is done, we cannot payout bug bounties due to the administrative overhead. All disclosures that are made until the launch, will later be processed under the same conditions, but we cannot guarantee when that happens.
+
+:::
+
```

Comment:
How does 

https://github.com/ory/kratos/security/policy

in particular "Supported Versions" fit into this here? I think we could use this document here also for the security policy in the open source?

Reply:
Hm this was mostly for Ory Cloud, but the same policy should apply to OS.
In any case, I think we need the SECURITY.md or GitHub complains, but we could copy its contents to this document and then just link the /docs in SECURITY.md

Supported Versions does not make a lot of sense in the Ory Cloud context, but if we explain it briefly (underlying core services etc.) it should work.

wdyt?